src/components/display/CollapsingActions.tsx

```diff
@@ -62,20 +62,16 @@
 
 		const visibleActions = useMemo(
 			() =>
-				map(
-					visibleItems,
-					({
-						iconType = 'ghost',
-						color = globalIconColor,
-						size = globalIconSize,
-						label,
-						...itemRest
-					}) => (
-						<Tooltip label={label} disabled={!label} key={itemRest.id}>
-							<IconButton {...itemRest} type={iconType} color={color} size={size} />
-						</Tooltip>
-					)
-				),
+				map(visibleItems, ({ label, ...itemRest }) => (
```

Comment:
We should keep extracting the iconType here, and set it in the type, as it was before, since iconType is not a valid IconButton prop